apps/microtvm/reference-vm/README.md

```diff
@@ -15,53 +15,86 @@
 
-### Organization
 
-Reference VMs are organized as follows:
+## Reference VMs Organization
+
+Reference VMs are organized in this directory as follows:
+
+```
+.
++-- base-box-tool.py - Reference VM build, test, and release tool.
++-- <platform>/      - One or more dirs related to the supported platform(s).
+    +-- Vagrantfile  - Vagrantfile that end-users will invoke. Should be based
+    |                  off a base box which contains dependencies other than the
+    |                  TVM python dependencies.
+    +-- base-box/    - Top-level directory which defines the base box.
+        +-- Vagrantfile.packer-template - 'packer' template Vagrantfile which
+        |                                 will be used to build the base box.
+        +-- test-config.json            - JSON file explaining how to perform
+                                          release tests to base-box-tool.py.
+```
 
-* `base-box-tool.py` - Reference VM build, test, and release tool
-* `<platform>/`
-** `Vagrantfile` Vagrantfile that end-users will inovke. Should be based off a base box
-    which contains dependencies other than the TVM python dependencies.
-** `base-box` - Top-level directory which defines the base box.
-*** `Vagrantfile.packer-template` - Packer template Vagrantfile which will be used to build the
-    base box.
-*** `test-config.json` - JSON file explaining how to perform release tests to `base-box-tool.py`
 
 ## Creating Releases
 
-1. Build the base box for the given platform: `$ ./base-box-tool.py [--provider=<provider>] build <platform>`
-2. Run release tests for each platform:
-    1. Connect any needed hardware to the VM host machine.
-    2. Run tests: `$ ./base-box-tool.py [--provider=<provider>] test [--microtvm-platform=<platform>] <platform> [--test-device-serial=<serial>]`. This
-       command does the following for each provider:
-        1. Copies all files inside `./<platform>` except `.vagrant` and `base-box` to
-           `./release-test`. This is done to avoid reusing any VM the developer may have started.
-        2. Executes `$ vagrant up [--provider=<provider>]`.
-        3. Finds an attached USB device matching the VID and PID specified in `test-config.json`,
-           and if `--test-device-serial` was given, that serial number (as reported to USB). Creates
-           a rule to autoconnect this device to the VM, and also attaches it to the VM>
-        4. SSHs to the VM, `cd` to the TVM root directory, and runs `test_cmd` from
-           `test-config.json`. Nonzero status means failure.
-3. If release tests fail, fix them and restart from step 1.
-4. If release tests pass: `$ ./base-box-tool.py [--provider=<provider>] release <--release-version=<version>> <--platform-version=<version>> <platform>`. Be sure you've logged
-   in to Vagrant Cloud using the `vagrant` tool.
+1. **Build** the base box for a given platform:
+```bash
+$ ./base-box-tool.py [--provider=<provider>] build <platform>
```

Comment:
A nit in command line terminology. Am I correct to assume `--provide` is optional and `PLATFORM` i mandatory?

```suggestion
$ ./base-box-tool.py [--provider=PROVIDER] build PLATFORM
```

Can you also put an example?

Reply:
@leandron Thanks a lot for the review!

> A nit in command line terminology. Am I correct to assume `--provide` is optional and `PLATFORM` i mandatory?

Yes, correct `--provide` is optional - if not specified all the available providers will be tried  (virtualbox, vmware, etc) - and PLATFORM is mandatory. I have changed the syntax in the doc as you suggested. 

> Can you also put an example?

Sure. Done in https://github.com/apache/tvm/pull/8613/commits/499427b720796641fabfe26eba792e6dac335e7e